framework/areg/base/private/Version.cpp

```diff
@@ -87,15 +89,24 @@ Version & Version::convFromString( const char * version )
     mMinor  = 0;
```

Comment:
The method ` Version & Version::convFromString( const char * version )` can be converted ` Version & Version::convFromString( const std::string & version )` 
What do think?

Reply:
The plan is to go through entire library to review the usage of const char* later on after all usages of String are converted to std::string